nixos/modules/misc/mandoc.nix

```
@@ -5,59 +5,192 @@ let
 
   cfg = config.documentation.man.mandoc;
 
-in {
+  toMandocOutput = output: lib.concatLines (
```

To fx the eval error :)

```suggestion
  toMandocOutput = output: (
```